### ISSUE TITLE
Property is not phpdoc'ed

### ISSUE THREAD

**Issue by rentalhost** [kalessil/phpinspectionsea#252]
This should inpect declared properties that doesn't declares a phpdoc. It currently happen with methods. Maybe related to #250.

```php
// Warning: $property don't specify a phpdoc.
public $property;
```
